### ISSUE TITLE
Crash on view SymLoadModuleEx function

### ISSUE THREAD

**Issue by maidamai0** [wolfpld/tracy#93]
I encountered a crash when I clicked `SymLoadModuleEx` function:
![image](https://user-images.githubusercontent.com/16829798/90212349-b16ba700-de25-11ea-90dc-b5e052da3fee.png)

OS: Windows 10 2004
Compiler:
```cmd
❯ cl
Microsoft (R) C/C++ Optimizing Compiler Version 19.27.29111 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.
```
Tracy version: [aa72cb6
](https://github.com/wolfpld/tracy/commit/aa72cb6ab9b546143a9513b8d956bb47276c69d5)

dump and stack backtrace:
[stack.txt](https://github.com/wolfpld/tracy/files/5072592/stack.txt)
[crash.dmp.zip](https://github.com/wolfpld/tracy/files/5072593/crash.dmp.zip)

Profiler and the profiled application run on the same machine.

**Comment by maidamai0** [comment]
crashed on this line [assert fail](https://github.com/wolfpld/tracy/blob/master/server/TracySourceView.cpp#L436)

**Comment by wolfpld** [comment]
Should be fixed in 5e559322.
